### PR TITLE
Add Japanese version and note about piano key re-rolls

### PR DESCRIPTION
**Pull request by katabame**
I added Japanese translate version of document, and wrote about piano key re-rolls.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,7 +55,8 @@ the piano in Scarface's office and hit the following keys in order.
 
 This will stop the device from rotating. Interacting with the device will reveal
 ciphered riddles. These turn out to be required achievements to enable the
-secret.
+secret.  
+**Note**: Every time you hit the piano keys, these required achievments list are re-rolled.
```
Comment (r-pufky) [inline README.md]:
```suggestion
**Note**: Every time you hit the piano keys, these required achievements list are re-rolled.
```

File: README_JA.md [hunk 2]
```diff
@@ -0,0 +1,479 @@
+Payday 2 シークレット
+---------------
+PD2のシークレットを解くための情報。
```
Comment (r-pufky) [inline README_JA.md:3]: I'd add a note to refer to the english doc. This doc potentially can get out of date.

**Review by r-pufky — COMMENTED** [review]
Awesome, thanks for the work!

fixed a spelling error, and requested that you leave a comment about this doc potentially getting out of date. Once those are resolved, I'll merge the pull request.

**Comment by katabame** [comment]
Thank you for the review, I fixed that spelling error and added language switcher. also added attention sentence for potentially getting out of date in Japanese version.
If I have to improve more, please notify me.🙌

**Comment by r-pufky** [comment]
Really appreciate all the work you put into this!

**Pull request merged**